### PR TITLE
Search ts21 26 error messages for post request

### PR DESCRIPTION
**Pull request by joshjames279**
Added JSON error messages to the controller for searches that don't exist and for use of non alphabetical symbols.

Updated the build.gradle to include a json altering package for the above purpose. Seen on the build.gradle, line 25.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -22,7 +22,7 @@ dependencies {
     implementation          'org.springframework.boot:spring-boot-starter-data-jpa'
     implementation          'org.flywaydb:flyway-core'
     runtimeOnly             'org.postgresql:postgresql'
-
+	implementation 			'org.json:json:20171018'
```
Comment (dearshrewdwit) [inline build.gradle:25]: Can you document why this is needed?

File: src/main/resources/data-postgres.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+INSERT INTO users(id, first_name, last_name, email, photo, education) VALUES(1, 'Ada', 'Lovelace', 'ada@lovelace.com', 'img', 'education');
```
Comment (dearshrewdwit) [inline src/main/resources/data-postgres.sql:1]: Can you remove this file - this should no longer be part of the app initialisation process

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
Nice stuff!

See inline comments

File: src/main/resources/data-postgres.sql (hunk 2 above)
Comment (joshjames279) [reply src/main/resources/data-postgres.sql]: Done

File: build.gradle (hunk 1 above)
Comment (joshjames279) [reply build.gradle:25]: Edited the comment for the pull request

Comment (joshjames279) [reply build.gradle:25]: Done

**Review by AntMousinho — APPROVED** [review]

**Review by dearshrewdwit — APPROVED** [review]

**Pull request merged**